### PR TITLE
Refresh price dinamically (via JS) on Swiper slide change

### LINKED ISSUE #10: Add a way to refresh the price
Once user switch to another product at carousel, price should be updated.

This can be done via:
- Endpoint
- JS?

### PR DESCRIPTION
**Pull request by igormukhingmailcom**
- Ended up there is no easy/acceptable way to calculate look price at controller based on currently selected product at each part
- So I've implemented Look's price calculation with JS, based on each Part's selected Product's price and Look's discount.

Fixes: #10

Todo:
- [x] We have different look price calculated via JS (`SUM(productPrice * look.discount)`) and via PHP (`look.discount * SUM(productPrice)`), I believe it should be changed to `SUM(productPrice * look.discount)` for PHP - it also will fix the possible issue at OrderProcessor which add discounts for every single product too.

### CODE REVIEW

**Review by loevgaard — APPROVED** [review]

File: src/Calculator/LookPriceCalculator.php [hunk 1]
```diff
@@ -27,30 +26,70 @@ public function __construct(
...
+    {
+        return $this->calculateCallableTotal($look, function (ProductVariantInterface $productVariant, array $context) use ($look): int {
+            $itemPrice = $this->productVariantPriceCalculator->calculate(
+                $productVariant,
+                $context
+            );
 
-        $channel = $context['channel'];
-        Assert::isInstanceOf($channel, ChannelInterface::class);
+//            dump('calculateDiscount', $itemPrice, $look->getDiscount(), (int) round($itemPrice * $look->getDiscount()));
```
Comment (loevgaard) [inline src/Calculator/LookPriceCalculator.php]:
```suggestion
```

**Review by loevgaard — APPROVED** [review]

**Comment by loevgaard** [comment]
@igormukhingmailcom: Shouldn't this be merged? :)

**Comment by igormukhingmailcom** [comment]
I will do final test on our app today and merge then

**Comment by loevgaard** [comment]
Thank you :)

**Comment by igormukhingmailcom** [comment]
I've checked and prices not updates on slide change, so I will try to fix this and merge after.

**Comment by loevgaard** [comment]
Can we merge this? :)

**Pull request merged**